src/shared/components/proteinChainPanel/ProteinChainPanel.tsx

```diff
@@ -221,6 +246,17 @@ export default class ProteinChainPanel extends React.Component<ProteinChainPanel
                     <div style={{
                         marginLeft:this.props.geneXOffset || 0,
                     }}>
+                        <div style={{display: this.isExpanded ? "inherit" : "none", position: "absolute", left: 20}}>PDB
+                            <DefaultTooltip
+                                placement="left"
+                                overlay={this.helpTooltipContent}
+                                arrowContent={<div className="rc-tooltip-arrow-inner"/>}
```

Comment:
actually we don't need the `arrowContent` anymore, it is already in the `DefaultTooltip` library